Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         .package(url: "https://github.com/yonaskolb/JSONUtilities.git", from: "4.2.0"),
         .package(url: "https://github.com/kylef/Spectre.git", from: "0.9.2"),
         .package(url: "https://github.com/onevcat/Rainbow.git", from: "3.0.0"),
-        .package(url: "https://github.com/tuist/XcodeProj.git", from: "8.9.0"),
+        .package(url: "https://github.com/BarredEwe/XcodeProj.git", .branch("feature/build-tool-plugin")), // Will be support 8.13.0
```

Comment:
Now, we need to wait until PR https://github.com/tuist/XcodeProj/pull/784 was pushed.